src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -62,10 +63,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       request.getRequestDispatcher("/WEB-INF/view/register.jsp").forward(request, response);
       return;
     }
-
-    User user = new User(UUID.randomUUID(), username, passwordHash, Instant.now());
-    userStore.addUser(user);
-
+    
+    ArrayList<String> whiteList = new ArrayList<String>();
```

Comment:
You should make this a static final class variable instead. This is so we don't have to initialize this every time this method is called. Something like the following where userStore is declared:

private static final ArrayList\<String> SUPER_USER_WHITE_LIST = new ArrayList\<String>(){{
  add("Jad");
  add("Trisha");
  add("Nathalia");
  add("Tyler");
  add("Han);
}};

(see https://beginnersbook.com/2013/12/how-to-initialize-an-arraylist/)